Tools/CISupport/ews-build/steps.py

```diff
@@ -653,9 +653,11 @@ class AnalyzeChange(buildstep.BuildStep):
 
     def _get_patch(self):
         sourcestamp = self.build.getSourceStamp(self.getProperty('codebase', ''))
-        if not sourcestamp or not sourcestamp.patch:
-            return None
-        return sourcestamp.patch[1]
+        if sourcestamp and sourcestamp.changes:
```

Comment:
This new approach is much better and way faster since it doesn't involve network request.